tools/sotn_str/jp.py

```diff
@@ -259,7 +264,7 @@ def dakuten_to_bytes(input_chr):
 
 
 def utf8_to_byte_literals(input_str):
-    clean_str = input_str.replace("_SJ(", "").replace(")", "")
+    clean_str = input_str.replace("_SJ(", "(")
```

Comment:
I dont' see any use (or definition) of the `_SJ` macro, but the second replacement on this line made it impossible to have a right parenthesis in an `_S` encoded string. The first replacement now replaces `_SJ(` with a left parens, so something like `_SJ("foo")` would become `("foo")`.

Reply:
Anything related to SJ can be removed, it's left over from when I thought there was a US encoding and a Japanese one